document-store-service/src/main/java/dk/dbc/search/solrdocstore/BibliographicBean.java

```diff
@@ -0,0 +1,44 @@
+package dk.dbc.search.solrdocstore;
+
+import dk.dbc.commons.jsonb.JSONBContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.ejb.Stateless;
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.UriInfo;
+
+@Stateless
+@Path("")
+public class BibliographicBean {
+    private static final Logger log = LoggerFactory.getLogger(BibliographicBean.class);
+
+    
+    private final JSONBContext jsonbContext = new JSONBContext();
+
+    @PersistenceContext(unitName = "solrDocumentStore_PU")
+    EntityManager entityManager;
+
+    @POST
+    @Path("bibliographic")
+    @Consumes({ MediaType.APPLICATION_JSON })
+    @Produces({MediaType.APPLICATION_JSON})
+    public Response addBibliographicKeys(@Context UriInfo uriInfo, String KeyJsonContent) throws Exception {
+        log.info("AddBibliographicKeys called ");
+        
+        BibliographicEntity be=jsonbContext.unmarshall(KeyJsonContent, BibliographicEntity.class);
+
+        entityManager.merge( be );
```

Comment:
Jeg kender ikke helt use-casen for addBibliographicKeys metoden - men hvis der er tale om SQL INSERTs only, så gætter jeg på at merge er en dyrere operation end persist, da den formodentlig først vil lave en SQL SELECT for at konstatere om der er tale om SQL INSERT eller SQL UPDATE?